### PR TITLE
Feature/export generic visit

### PR DESCRIPTION
**Pull request by joswarmer**
All tests in validation are now in test and mocha-ified. Branch name makes no sense, please ignore.

### CODE REVIEW

**Comment by dslmeinte** [comment]
Need to merge https://github.com/LionWeb-io/lionweb-integration-testing/pull/19 before we can merge this, I guess.

File: packages/test/src/m3/ecore/importer.test.ts [hunk 1]
```diff
@@ -22,6 +22,8 @@ describe("Ecore importer", () => {
             logIssues(issues)
             deepEqual(issues, [])
             const serialization = serializeLanguages(language)
+            // console.log("original\n" + JSON.stringify(sortedSerialization(serialization)))
```
Comment (dslmeinte) [inline packages/test/src/m3/ecore/importer.test.ts:25]: Can we remove this?

File: packages/test/.mocharc.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (dslmeinte) [inline packages/test/.mocharc.json:1]: Using `ts-node` works? Does this then necessitate changes in `package.json`, i.e. a dependency + changing how running the tests is invoked?

File: packages/validation/src/json/LanguageUtils.ts [hunk 3]
```diff
@@ -9,4 +12,45 @@ export class LanguageWrapper extends LionWebJsonChunkWrapper {
     constructor(languageJson: LionWebJsonChunk) {
         super(languageJson);
     }
+    
+    // inheritsFrom = (classifier: LionWebJsonNode): LionWebJsonNode[] => {
```
Comment (dslmeinte) [inline packages/validation/src/json/LanguageUtils.ts:16]: Would switching on https://github.com/cartant/eslint-plugin-etc/blob/main/docs/rules/no-commented-out-code.md help to detect commented-out code?

**Review by dslmeinte — CHANGES_REQUESTED** [review]

File: packages/validation/src/json/LanguageUtils.ts (hunk 3 above)
Comment (joswarmer) [reply packages/validation/src/json/LanguageUtils.ts:16]: Maybe, but just searching for // will also be quick.

File: packages/test/src/m3/ecore/importer.test.ts (hunk 1 above)
Comment (joswarmer) [reply packages/test/src/m3/ecore/importer.test.ts:25]: Done, was for debugging some stuff.

File: packages/test/.mocharc.json (hunk 2 above)
Comment (joswarmer) [reply packages/test/.mocharc.json:1]: Removed all the ts-node stuff, testing still works.

File: packages/validation/src/json/LanguageUtils.ts (hunk 3 above)
Comment (joswarmer) [reply packages/validation/src/json/LanguageUtils.ts:16]: LanguageWrapper is unfinished work and LionwebDiff is very outdated.  So we can either leave it as is, or remove them completely. What do you think?

File: packages/test/.mocharc.json (hunk 2 above)
Comment (joswarmer) [reply packages/test/.mocharc.json:1]: After changing package.json and doing install I get `Couldn't find any versions for "@lionweb/core" that matches "0.5.0-beta.9"`

Comment (joswarmer) [reply packages/test/.mocharc.json:1]: > After changing package.json and doing install I get `Couldn't find any versions for "@lionweb/core" that matches "0.5.0-beta.9"`

Solved now, probably something old hanging around.

File: packages/validation/src/json/LanguageUtils.ts (hunk 3 above)
Comment (dslmeinte) [reply packages/validation/src/json/LanguageUtils.ts:16]: I'm very much in favor of removing unfinished work(/"stuff").

**Comment by joswarmer** [comment]
> Need to merge [LionWeb-io/lionweb-integration-testing#19](https://github.com/LionWeb-io/lionweb-integration-testing/pull/19) before we can merge this, I guess.

Has been done I see

**Pull request merged**